http/headers/Link.json

```
@@ -32,42 +32,6 @@
             "deprecated": false
           }
         },
-        "blocking": {
-          "__compat": {
-            "spec_url": "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-blocking",
-            "tags": [
-              "web-features:blocking-render"
-            ],
-            "support": {
-              "chrome": {
-                "version_added": "105"
```

To make this easier to comprehend, and based on https://github.com/chromium/chromium/commit/1d8c45ad4c0816c6b4a21beb5741a8003ea40fe0#diff-4b938e3a1dedc77093c5a2e3471fb2cb1c9c89e953fefc2394d9b27b65f8c91d and https://github.com/chromium/chromium/commit/1372f2379a46b86177b5537b5be77a00c83c6bda, could we first update this statement in the next release, and delete the statement only in the following release:

```suggestion
                "version_added": "103",
                "version_removed": "105"
```

/cc @ddbeck

---

Yeah, it'd be nice for the blame to show this—so there's two commits for it. But it does meet the guidelines, so it's OK if it all happens in the same _release_.

---

Thanks, Daniel. Let's do this then!